### ISSUE TITLE
Missing translations

### ISSUE THREAD

**Issue by standus** [PhpUnitsOfMeasure/php-units-of-measure#60]
Hi,
nice library. 
By me, very usefull will be locatizations of units.
For example:
l => en=litre, de=liter, etc...

**Comment by stevergill** [comment]
I agree, this would be very useful.

**Comment by triplepoint** [comment]
I agree, and I've got some preliminary work done on extracting a localization file, but it might be a little while before it's complete.

I know it's not very convenient, but in the meantime you can add aliases at run-time to the appropriate units of measure.  See the [readme section on runtime aliasing](https://github.com/PhpUnitsOfMeasure/php-units-of-measure/#adding-new-aliases-to-existing-units).

I'll leave this ticket open and comment here when there's more on this subject.
